blimp-live-widgets/src/components/Weather/index.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import styles from './styles.css'
+
+export default class Weather extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      weather: {
+        city: "Waterloo",
+        temperature: 0,
+        main: "None",
+        description: "None",
+      },
+      isLoaded: false
+    };
+  }
+
+   componentDidMount() {
+    fetch("http://api.openweathermap.org/data/2.5/weather?q=Waterloo,CA&appid=d79a0d378573a8b844d4a8b7e1c963d6")
```

Comment:
can you just add a comment/a todo that we should do something with this like maybe encrypt it but don't have to deal with it now.